etna/packages/etna-js/components/metis_exploration.tsx

```diff
@@ -146,42 +163,82 @@ function FileOrFolderInner({ optionSet, path, target, setTarget, onEmpty, placeh
   />
 }
 
-export function PickFileOrFolder({ project_name=CONFIG.project_name, bucket, setPath, path, allowFiles=true, label, basePath, topLevelPlaceholder='', className}: {
+export function PickFileOrFolder({ project_name=CONFIG.project_name, bucket, setPath, path, useTargetType, allowFiles=true, label, basePath, topLevelPlaceholder='', className, disablePortal=true}: {
```

Comment:
Probably out of scope for this PR, but I tried adding a new file to a bucket after I already selected a file, and it didn't show up. Maybe add a refresh button somewhere in the input?

Reply:
fair!  Refreshing the page achieves this, but there's machinery built in to where this may not be hard to add in to the component itself so that you can refresh the `contentsSeen` without loosing all the other intermediate state that's lost with a page referesh.